platforms/Windows/Directory.Build.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <ProductArchitecture Condition=" '$(ProductArchitecture)' == '' ">amd64</ProductArchitecture>
     <ProductVersion Condition=" '$(ProductVersion)' == '' ">0.0.0</ProductVersion>
+    <InstallerPlatform Condition=" '$(InstallerPlatform)' == '' ">x86</InstallerPlatform>
```

Comment:
I think that we should be setting this at the SDK level, not at the global level. This would otherwise install the ARM64 slice of the runtime library into the wrong architecture.

Reply:
Following offline discussion, this should be set by the command line invocation. Closing.